src/operator/tensor/broadcast_reduce_op.h

```diff
@@ -977,24 +1037,40 @@ inline void BroadcastComputeImpl(const nnvm::NodeAttrs& attrs,
                                  const mxnet::TShape& small) {
   using namespace mshadow;
   using namespace mshadow::expr;
+  using namespace mxnet_op;
   mxnet::TShape src_shape, dst_shape;
   BroadcastReduceShapeCompact(outputs[0].shape_, small, &dst_shape, &src_shape);
   Stream<xpu> *s = ctx.get_stream<xpu>();
```

Comment:
can we name the variable "stream" instead of single letter?

Reply:
I'm also in favor of a better name than `s` but considering the fact that:
1. This name `s` is used in most of the existing operator code (also the same as the current operator implementation tutorial)
2. A specific style improvement to existing code is orthogonal with this PR's major target (adding the infrastructure for Numpy-compatible project).
I'm not in favor of changing this variable name here.